ngx-tools/src/verify-types/array-has-all-elements-set/array-has-all-elements-set.ts

```diff
@@ -7,7 +7,7 @@ export function arrayHasAllElementsSet<A, B, C, D>(
   arr: [A | undefined | null, B | undefined | null, C | undefined | null, D | undefined | null],
 ): arr is [A, B, C, D];
 
-export function arrayHasAllElementsSet(arr: any): boolean {
+export function arrayHasAllElementsSet(arr: any): arr is Array<any> {
```

Comment:
I'm glad you added this any.. I wasn't thinking about the inner types originally, but looking at this now, we should probably allow the consumer to define the type inside the array.

```typescript
export function arrayHasAllElementsSet<T>(arr: any): arr is Array<T> {
```

That way it doesn't always have to be `any`.